environments/terraform-example/.gitignore

```diff
@@ -0,0 +1 @@
+inventory/hosts
```

Comment:
What's the basis for gitignoring this? Just that it's really controlled by terraform?